lib/node_modules/@stdlib/math/base/special/asinf/docs/types/index.d.ts

```diff
@@ -0,0 +1,48 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/**
+* Computes the arcsine of a single-precision floating-point number.
+*
+* @param x - input value
+* @returns arcsine (in radians)
+*
+* @example
+* var v = asinf( 0.0 );
+* // returns ~0.0
```

Comment:
```suggestion
* // returns 0.0
```

In the README, we have this as exact. I'd expect the same here.